tools/onnx2ncnn_quant_table.py

```diff
@@ -44,16 +47,10 @@ def get_table(onnx_path: str,
     # get an available input shape randomly
     for _, input_data in enumerate(dataloader):
         input_data = data_preprocessor(input_data)
-        input_tensor = input_data[0]
-        if isinstance(input_tensor, list):
-            input_shape = input_tensor[0].shape
-            collate_fn = lambda x: data_preprocessor(x[0])[0].to(  # noqa: E731
-                device)
-        else:
-            input_shape = input_tensor.shape
-            collate_fn = lambda x: data_preprocessor(x)[0].to(  # noqa: E731
-                device)
-        break
+        input_tensor = input_data['inputs']
```

Comment:
In some old testcase, `input_tensor` is `list`.

Reply:
@irexyc Did we only test on mmcls models? Could we test on models from other codebases as well?

Reply:
tested ok on mmpose.